### ISSUE TITLE
Can not execute the example code.

### ISSUE THREAD

**Issue by kym0802** [WenjieDu/PyPOTS#291]
### 1. System Info

PyPots 0.2.1
Python 3.11.5
Windows 11 64bit

### 2. Information

- [X] The official example scripts
- [ ] My own created scripts

### 3. Reproduction

'# It is PyPots basic example on the Github page

import numpy as np
from sklearn.preprocessing import StandardScaler
from pygrinder            import mcar
from pypots.data          import load_specific_dataset
from pypots.imputation    import SAITS
from pypots.utils.metrics import cal_mae
'# here I corrected calc_mae to cal_mae

'# Data preprocessing. Tedious, but PyPOTS can help.
data = load_specific_dataset('physionet_2012')  # PyPOTS will automatically download and extract it.

X = data['X']
num_samples = len(X['RecordID'].unique())
X = X.drop(['RecordID', 'Time'], axis = 1)
X = StandardScaler().fit_transform(X.to_numpy())
X = X.reshape(num_samples, 48, -1)

X_ori = X             # keep X_ori for validation
X     = mcar(X, 0.1)  # randomly hold out 10% observed values as ground truth
dataset = {"X": X}    # X for model input
print(X.shape)  # (11988, 48, 37), 11988 samples, 48 time steps, 37 features

'# Model training. This is PyPOTS showtime.
saits = SAITS(n_steps=48, n_features=37, n_layers=2, d_model=256, d_inner=128, n_heads=4, d_k=64, d_v=64, dropout=0.1, epochs=10)

'# Here I use the whole dataset as the training set because ground truth is not visible to the model, you can also split it into train/val/test sets
saits.fit(dataset)

### 4. Expected behavior

I did no modification to the basic example but I can not execute it.

![image](https://github.com/WenjieDu/PyPOTS/assets/21983055/4f76ffc6-1d48-4af8-b717-c909116d6006)

**Comment by WenjieDu** [comment]
Hi there 👋,

Thank you so much for your attention to PyPOTS! You can follow me on GitHub to receive the latest news of PyPOTS. If you find PyPOTS helpful to your work, please star⭐️ this repository. Your star is your recognition, which can help more people notice PyPOTS and grow PyPOTS community. It matters and is definitely a kind of contribution to the community.

I have received your message and will respond ASAP. Thank you for your patience! 😃

Best,
Wenjie

**Comment by WenjieDu** [comment]
Hi Youngmin @kym0802, which version of PyGrinder you're using?

**Comment by kym0802** [comment]
Hi Weijie. My PyGrinder version is 0.4. I updated it now, and shows it again.
I installed PyGrinder through Conda. Could it be a problem?

**Comment by WenjieDu** [comment]
Try upgrading your PyPOTS version to `0.3`. If the error still exists, please paste the whole traceback log here.
